src/diffusers/pipelines/audioldm/pipeline_audioldm.py

```diff
@@ -340,7 +340,6 @@ def check_inputs(
                     f" {negative_prompt_embeds.shape}."
                 )
 
-    # Copied from diffusers.pipelines.stable_diffusion.pipeline_stable_diffusion.StableDiffusionPipeline.prepare_latents with width->self.vocoder.config.model_in_dim
```

Comment:
I used this `# Copied` from assuming that we don't check that function arguments match, only the main body of the function, so I removed the `width` argument without any problem

If you look at the preceding line, you see that I set: https://github.com/huggingface/diffusers/blob/2bfa55f4ed39f9dd6fed82720d87f6d4e60f21e3/src/diffusers/pipelines/audioldm/pipeline_audioldm.py#L343

=> so I copied the main body of the function, but replacing `width` with `self.vocoder.config.model_in_dim`. We didn’t check that the function arguments matched, so I could just remove `width` without any problem and have it default to the vocoder attribute.

We have two options here to update this:
1. Remove the `# Copied from` for this method
2. Keep the `# Copied from`, but add the `width` argument back into the arguments, and pass` width=self.vocoder.config.model_in_dim` when we call the method

Happy to open a PR to fix this if we want to revisit these changes